docs/src/GettingStartedPage.js

```diff
@@ -21,7 +21,12 @@ const Page = React.createClass({
                   <h2 id="setup" className="page-header">Setup</h2>
                   <p className="lead">You can import the lib as AMD modules, CommonJS modules, or as a global JS script.</p>
 
-                  <p>First add the Bootstrap CSS to your project; check <a href="http://getbootstrap.com/getting-started/" name="Bootstrap Docs">here</a> if you have not already done that. Then:</p>
+                  <p>First add the Bootstrap CSS to your project; check <a href="http://getbootstrap.com/getting-started/" name="Bootstrap Docs">here</a> if you have not already done that. Note that:</p>
+                  <ul>
+                    <li>Because many folks use custom bootstrap themes, we do not directly depend on Bootstrap. It is up to you to to determine how you get and link to the bootstrap css and fonts.</li>
+                    <li>React-Bootstrap doesn't really depend on any bootstrap version (unless there are big breaking changes in the main lib, which are pretty rare). Just have bootstrap in your dependencies and ensure your build can read your less/sass/scss entry point.</li>
```

Comment:
@mtscout6 okay, was just paraphrasing @dozoisch from https://github.com/react-bootstrap/react-bootstrap/issues/487#issuecomment-96770300 . Here's an updated PR that hopefully works better.